s4e-web/cypress/integration/settings/settings-institution-people.spec.ts

```diff
@@ -6,64 +6,216 @@ import {SettingsInstitutions} from '../../page-objects/settings/settings-institu
 import {SettingsInstitutionPeople} from '../../page-objects/settings/settings-institution-people.po';
 import {ConfirmModal} from '../../page-objects/modal/confirm-modal.po';
 import {SettingsNav} from '../../page-objects/settings/settings-navigation.po';
+import {Registration} from '../../page-objects/auth/auth-register.po';
+import {SettingsUserDeleteAccount} from '../../page-objects/settings/settings-delete-account.po';
+import {UserOptionsAuthentication} from '../../page-objects/user-options/user-option-authentication.po';
 
 before(() => {
   cy.fixture('users/zkAdmin.json').as('zkAdmin');
   cy.fixture('users/zkMember.json').as('zkMember');
+  cy.fixture('users/userToRegister.json').as('userToRegister');
 });
 
 describe('Inviting people', () => {
   beforeEach(function () {
     cy.server();
     cy.visit('/login');
-    Login.loginAs(this.zkAdmin);
   });
 
-  it('should invite, re-invite and remove person from the institution', function () {
-    UserOptionsGoToSettings.gotoUserProfile();
-    SettingsInstitutions.selectNthInstitution(0);
-    SettingsInstitutionPeople.goToPeopleInInstitutionPage()
-      .addInvitation('test@mail.pl')
-      .invitationWithEmailShouldExist('test@mail.pl')
-      .resendToNewEmail('test@mail.pl')
-      .removeFromInstitution('test@mail.pl');
-    ConfirmModal.accept();
+  context('Accept invitation to institution', () => {
```

Comment:
I'm considering simplification of the tests by doing calls to API, in case when feature have been checked already. Maybe it isn't required for this PR, but I'd make ticket for further e2e tests maintenance.

For example if you want:
- Receive actual members of institution, call GET /institutions/{institution}/members
- remove an invited people from institution, call DELETE /institutions/{institution}/members/{appUserId}

...

Reply:
Swagger: http://localhost:4201/swagger-ui/index.html?configUrl=/v3/api-docs/swagger-config

should display persons
	cy.req(GET, '/api/v1/institutions/{institution}/members')
  	.should('contain', email')
  	.as('persons');
  goToList()
  	.should('contain', @persons);
  personIsAdmin.should('have', '.edition-btn');

should send invitation to person
  invite(email, is_admin)
  checkEmailBy(subject)
  
should re-send invitation to person
  invite(email, is_admin)
  checkEmailBy(subject)
  
should add invited person into an institution
  cy.req(POST, '/api/v1/institutions/{institution}/members', {email: 'test'});
  clickActivationUrl();
  register();
  cy.req(GET, '/api/v1/institutions/{institution}/members')
  	.should('contain', email');
  	
  // check if is admin
  
should add invited person with admin privileges into an institution
  cy.req(POST, '/api/v1/institutions/{institution}/members', {email: 'test'});
  clickActivationUrl();
  register();
  cy.req(GET, '/api/v1/institutions/{institution}/members')
  	.should('contain', email');
  	
  // check if is admin